jsf/plugins/org.eclipse.jst.pagedesigner/src/org/eclipse/jst/pagedesigner/preview/PageExpressionContext.java

```diff
@@ -112,7 +112,7 @@ public void popPageVarProvider(IPageVariablesProvider provider) {
 	 */
 	public Object evaluateExpression(String expression, Class expectedClass, Map options)
 			throws ELException {
-		//Bug 319317 - Third-party plug-in providing javax.servlet.jsp.el version 2.1 or greater breaks WPE preview
+		//Bug 319317 - Third-party plug-in providing jakarta.servlet.jsp.el version 2.1 or greater breaks WPE preview
```

Comment:
Revert this change. The `javax` is part of the title of the issue.

Reply:
change is [reverted](https://github.com/eclipse-jsf/webtools.jsf/pull/9/commits/4b40446e621ee49ab4464eb0eee9e399dd144f9c).